decidim-proposals/spec/services/decidim/proposals/searchable_proposal_rsrc_spec.rb

```diff
@@ -67,6 +67,7 @@ module Decidim
               proposal.update(title: updated_title)
 
               proposal.save!
+              searchable.reload
```

Comment:
Are you sure this solves the issue? Searchable isn't used again, so this shouldn't impact the test. Maybe I'm missing something, though 🤷🏽‍♂️

Reply:
Hi @josepjaume , the idea I'm trying to avoid is: if the first SearchableResource has locale :ca and then the first `SearchableResource` also is for locale :ca, then Rails will use the former cached instance instead of reloading the modified version in the database. This `reload` will avoid stale data.